framework/src/actions/ActionWarehouse.C

```diff
@@ -420,3 +427,23 @@ ActionWarehouse::getCurrentActionName() const
 {
   return (*_act_iter)->parameters().blockFullpath();
 }
+
+std::set<std::string>
+ActionWarehouse::meshTasks()
+{
+  std::set<std::string> tasks = {"setup_mesh",
+                                 "add_mesh_generator",
+                                 "execute_mesh_generators",
+                                 "set_mesh_base",
+                                 "add_partitioner",
+                                 "add_geometric_rm",
+                                 "attach_geometric_rm",
+                                 "init_mesh",
+                                 "prepare_mesh",
+                                 "add_mesh_modifier",
+                                 "execute_mesh_modifiers",
+                                 "add_mortar_interface",
+                                 "uniform_refine_mesh",
+                                 "setup_mesh_complete"};
+  return tasks;
```

Comment:
This isn't the way we do things anymore. The issue is that as we add or change the main list, we have to be careful to update this list. We explicitly dropped this way of doing things. Instead you can add a task and designate it as the "final" task. Then start executing the main list normally. This is how we handle the "mesh-only" option now.

Reply:
The problem here is that I need to skip tasks in between but not skipping tasks after a certain task. Setting `final` seems not a solution, right?

Reply:
Well Yuck... What tasks do you need to skip? Perhaps we could add a method to skip tasks too? I really would prefer to avoid the second list thing.

Reply:
I'm adding a new task in the early part of the simulation right now!

Reply:
I will wait on that.